webapp/app/templates/basis/base_step_form.html

```diff
@@ -39,4 +39,50 @@
         $('input[type="date"]').attr('max', '9999-12-31');
     </script>
     <script src="{{ url_for('static', filename='js/jquery.mask.min.js') }}" type=text/javascript></script>
+    <script>
+            $(document).ready(function(){
+                $('.date_input').mask('00.00.0000');
+                $('input[name="unlock_code"]').mask('A', {
+                    translation: {
+                        "A": { pattern: /[\w]/, recursive: true }
+                    }
+                });
+                window.memo_values  = new Array();
```

Comment:
This isn't being used.

Reply:
This script was added by mistake (s. below)